### PR TITLE
welcome format help

### PR DESCRIPTION
**Pull request by iiiiii1wepfj**

### CODE REVIEW

File: locales/en-GB/welcome.json [hunk 1]
```diff
@@ -9,5 +9,6 @@
...
+  "welcome_set_success": "The welcome message for {chat_title} were set successfully.",
+  "welcome_format_help_msg": "The message can be formatted in html format or with Telegram's builtin text formatter. You can use the /html command to verify that your html is correct.\n\nYou can use these arguments for the welcome message:\n - <code>{id}</code> to show the user ID in the welcome message.\n - <code>{username}</code> to show the user's username in the welcome message. if the user has no username the bot shows a mention.\n - <code>{first_name}</code> to show the user's name in the welcome message.\n - <code>{full_name}</code> or <code>{name}</code> to show the full name in the welcome message use the argument.\n - <code>{mention}</code> to mention the user in the welcome message.\n - <code>{chat_title}</code> or <code>{title}</code> to show the chat title in the welcome message.\n\nTo add buttons you can use <code>[button text](buttonurl:button link)</code>. To add more buttons on the same line, you can add :same after the link, like this: <code>[button text](buttonurl:button link:same)</code>."
```
Comment (AndrielFR) [inline locales/en-GB/welcome.json:13]: `\n - <code>` -> `\n- <code>`

**Review by AndrielFR — CHANGES_REQUESTED** [review]

File: locales/en-GB/welcome.json (hunk 1 above)
Comment (alissonlauffer) [reply locales/en-GB/welcome.json:13]: That space is intended because it is a list.

Comment (AndrielFR) [reply locales/en-GB/welcome.json:13]: Hmmm.

**Review by AndrielFR — APPROVED** [review]

**Comment by iiiiii1wepfj** [comment]
@alissonlauffer this is good now?

**Review by alissonlauffer — APPROVED** [review]

**Comment by iiiiii1wepfj** [comment]
so can you merge the pr now?

File: eduu/plugins/rules.py [hunk 2]
```diff
@@ -74,6 +74,8 @@ async def show_rules_pvt(c: Client, m: Message, strings):
     else:
         await m.reply_text(strings("rules_empty"))
 
+    await m.stop_propagation()
```
Comment (AndrielFR) [inline eduu/plugins/rules.py:77]: `await` is not needed.

File: eduu/plugins/welcome.py [hunk 3]
```diff
@@ -32,11 +32,15 @@ def toggle_welcome(chat_id: int, mode: bool):
...
 
-@Client.on_message(filters.command(["welcomeformat", "start welcome_format_help"]))
+@Client.on_message(
+    filters.command(["welcomeformat", "start welcome_format_help"], prefix)
+)
 @use_chat_lang()
 async def welcome_format_message_help(c: Client, m: Message, strings):
     await m.reply_text(strings("welcome_format_help_msg"))
 
+    await m.stop_propagation()
```
Comment (AndrielFR) [inline eduu/plugins/welcome.py:42]: `await` is not needed.

File: eduu/plugins/start.py [hunk 4]
```diff
@@ -17,7 +17,8 @@
 from eduu.utils.localization import use_chat_lang
 
 
-@Client.on_message(filters.command("start", prefix) & ~filters.regex("^/start rules"))
+# Using a low priority group so deeplinks will run before this and stop the propagation.
+@Client.on_message(filters.command("start", prefix), group=2)
```
Comment (AndrielFR) [inline eduu/plugins/start.py:21]: Nah, because it is in another group, the `stop_propagation` of the other function will have no effect here.

Comment (iiiiii1wepfj) [reply eduu/plugins/start.py:21]: @AndrielFR hmm

Comment (AndrielFR) [reply eduu/plugins/start.py:21]: https://github.com/pyrogram/pyrogram/blob/master/pyrogram/dispatcher.py#L233 hmm, it seems that I was wrong. 🤔

**Pull request merged**